homeassistant/components/kodi/media_player.py

```diff
@@ -7,6 +7,11 @@
 import urllib
 
 import aiohttp
+
```

Comment:
remove that empty line

Reply:
Done

Reply:
Because that is wrong, if you use isort, you well see the setup -> https://developers.home-assistant.io/docs/en/next/development_guidelines.html